src/pytorch_lightning/cli.py

```diff
@@ -436,6 +436,7 @@ def subcommands() -> Dict[str, Set[str]]:
 
     def _add_subcommands(self, parser: LightningArgumentParser, **kwargs: Any) -> None:
         """Adds subcommands to the input parser."""
+        self.subcommand_parsers: Dict[str, LightningArgumentParser] = {}
```

Comment:
Could this be private?

Reply:
I agree with adding the underscore prefix

Reply:
I added the underscore prefix.